doc/architectural/cbmc-architecture.md

```diff
@@ -61,7 +61,7 @@ transformation tools (see \ref other-tools).
 
 For an explanation of the data structures involved in the modeling of a GOTO
 program (the GOTO Intermediate Representation used by CBMC and assorted tools)
-please see \subpage central-data-structures .
```

Comment:
:information_source: Removing this use of `\subpage` means that the Central Data Structures page is no longer considered a sub page of the architecture page which moves it to the root level in the navigation tree.
![image](https://user-images.githubusercontent.com/35029362/223410657-fe663e8c-9add-4a86-8c56-a78a075ceebc.png)